<?xml version="1.0"?>

        <!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
        <!--                                                                                    -->
        <!--                                                                                    -->
        <!--                            LAUNCH SENSOR PLACEMENTS MAP                            -->
        <!--                                                                                    -->
        <!--                                                                                    -->
        <!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
        <!--                              @AsifArain, Dec 08, 2017                              -->
        <!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->

<launch>


    <!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
    <!--                              environment map                                       -->
    <!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
	<node pkg="plan_execution" type="env_map_publisher_node" name="env_map_publisher" output="screen">
	
	    <param name="map_file"	        value="prismaforum5_map_conf.dat"/>
        <param name="map_size_file"	    value="prismaforum5_mapsize_conf.dat"/>
        <param name="cell_size_file"	value="prismaforum5_cellsize_conf.dat"/>
        <param name="robot_origin_file"	value="prismaforum5_origin_conf.dat"/>        
        <!--param name="conf_file"	        value="robot_plan_detection_test.dat"/-->
        
        <param name="hard_offset_x"	    value="0.0"/> <!--"0.0"-->
        <param name="hard_offset_y"	    value="0.0"/> <!--"-0.25"-->
        
        
    </node>
    
    
    <!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
    <!--                              confs placements                                      -->
    <!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
    <node pkg="plan_execution" type="conf_publisher_node" name="conf_publisher" output="screen">
	    
        <param name="cell_size_file"	value="prismaforum5_cellsize_conf.dat"/>
        <param name="robot_origin_file"	value="prismaforum5_origin_conf.dat"/>        
        <param name="conf_file"	        value="robot_plan_detection.dat"/>
        
        <param name="hard_offset_x"	    value="0.0"/> <!--"0.0"-->
        <param name="hard_offset_y"	    value="0.0"/> <!--"-0.25"-->
        
    </node>
    
</launch>


